src/PIL/TiffImagePlugin.py

```diff
@@ -1845,7 +1845,7 @@ def _save(im, fp, filename):
         e.setimage(im.im, (0, 0) + im.size)
         while True:
             # undone, change to self.decodermaxblock:
-            l, s, d = e.encode(16 * 1024)
+            l, s, d = e.encode(16 * 1024)  # noqa: E741
```

Comment:
Please can you check what `l` represents? Is there a better name?

Reply:
`l`, `s`, and `d` are the length of the encoded data, an error code, and the encoded data.